OtripleS.Web.Api/Services/Foundations/Teachers/TeacherService.cs

```diff
@@ -47,34 +50,31 @@ public partial class TeacherService : ITeacherService
 
             ValidateStorageTeacher(maybeTeacher, teacherId);
 
-            return await this.storageBroker.DeleteTeacherAsync(maybeTeacher);
+            return maybeTeacher;
         });
 
         public ValueTask<Teacher> ModifyTeacherAsync(Teacher teacher) =>
         TryCatch(async () =>
         {
-            ValidateTeacherOnModify(teacher);
-            Teacher maybeTeacher = await this.storageBroker.SelectTeacherByIdAsync(teacher.Id);
-            ValidateStorageTeacher(maybeTeacher, teacher.Id);
-            ValidateAgainstStorageTeacherOnModify(inputTeacher: teacher, storageTeacher: maybeTeacher);
+             ValidateTeacherOnModify(teacher);
```

Comment:
This needs correction
<img width="121" alt="image" src="https://user-images.githubusercontent.com/89320816/160246145-5470f358-8fab-480a-9e04-b94aca82e96e.png">

Reply:
@ShriHumrudha Conflict is resolved